api-report/container-runtime.api.md

```diff
@@ -240,14 +245,31 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents
     uploadBlob(blob: ArrayBufferLike, signal?: AbortSignal): Promise<IFluidHandle<ArrayBufferLike>>;
 }
 
+// Warning: (ae-forgotten-export) The symbol "IDataStoreAliasMessage" needs to be exported by the entry point index.d.ts
+//
+// @public (undocumented)
+export type ContainerRuntimeAliasMessage = ContainerRuntimeMessage<ContainerMessageType.Alias, IDataStoreAliasMessage>;
+
+// @public (undocumented)
+export type ContainerRuntimeBlobAttachMessage = ContainerRuntimeMessage<ContainerMessageType.BlobAttach, undefined>;
+
+// @public (undocumented)
+export type ContainerRuntimeChunkedOpMessage = ContainerRuntimeMessage<ContainerMessageType.ChunkedOp, IChunkedOp>;
+
+// @public (undocumented)
+export type ContainerRuntimeDataStoreOpMessage = ContainerRuntimeMessage<ContainerMessageType.FluidDataStoreOp, IEnvelope>;
```

Comment:
Do we need to export these?  Probably ties down to types exported below.  Would be good to take a pass and see if we can _stop_ exporting as much as possible here, even `ContainerRuntimeMessage` - not sure why it's there.

Reply:
Good question. I assumed that we do but if we can not export any of it that'd be awesome. I'll try it and see what happens.

Reply:
Cool.  I think you'll have to deprecate `ContainerRuntimeMessage` as-is and give the generic one a new name, right?

Reply:
Hmm, I suppose so. Now that you point it out, I'm actually surprised we can make the change as it currently is - adding required generic parameters - without breaking back-compat tests.

Reply:
Looks like there's a tinylicious test that uses `ContainerRuntimeMessage` (sigh). But that's it, lol.

Reply:
@noencke , where is the tinylicious use? Search tells me there are no such uses.

Reply:
My bad, I meant `ContainerMessageType`, not `ContainerRuntimeMessage`. I reduced the exports to the minimal without having to mess with back-compat tests.